scripts/globals/items/recall_ring_jugner.lua

```diff
@@ -0,0 +1,24 @@
+-----------------------------------
+-- ID: 15841
+-- Recall ring: Jugner
+-- Enchantment: "Recall-Jugner"
+-----------------------------------
+require("scripts/globals/status")
+require("scripts/globals/keyitems")
+require("scripts/globals/teleports")
+-----------------------------------
+local item_object = {}
+
+item_object.onItemCheck = function(target)
+    local result = 0
+    if target:hasKeyItem(xi.ki.JUGNER_GATE_CRYSTAL) == false then
+       result = 445
```

Comment:
This (and similar) conditionals could be simplified to `if not target:hasKeyItem(xi.ki.JUGNER_GATE_CRYSTAL) then`

Reply:
You got it boss! :)